website/docs/reference/dbt-jinja-functions/config.md

```diff
@@ -63,9 +66,76 @@ __Args__:
 
 The `config.require` function is used to get configurations for a model from the end-user. Configs defined using this function are required, and failure to provide them will result in a compilation error.
 
+:::warning Deprecation warning for meta fallback
```

Comment:
Should this be a reusable or are they different?

Reply:
There is a slight difference between them